src/cohort.js

```diff
@@ -0,0 +1,62 @@
+import { Student } from './student.js'
+
+class Cohort {
+  constructor() {
+    this.cohortList = []
+    this.studentsList = []
+  }
+
+  createCohort(cohortName) {
+    if (typeof cohortName !== 'string')
+      throw new Error('cohort name must be a valid string')
+    const newCohort = { name: cohortName, students: [] }
+    this.cohortList.push(newCohort)
+    return true
+  }
```

Comment:
Usually when performing creation operations, the function should return the created entity